docs/manual/developer/02_building_complianceascode.md

```diff
@@ -446,6 +446,16 @@ ctest -LE quick -j4
 
 Note: CTest does not run [SSG Test Suite](https://github.com/ComplianceAsCode/content/tree/master/tests) which provides simple system of test scenarios for testing profiles and rule remediations.
 
+## Profiling the buildsystem
+
+To make sure your changes don't prolong the time that it takes to build products by using the `build_product` script too much,
```

Comment:
We now prefer style "1 line = 1 sentence", because people won't have to re-wrap when modyfing the text.